src/components/NewSpendingForm/NewSpendingForm.jsx

```diff
@@ -69,10 +87,23 @@ const NewSpendingForm = ({ onClose }) => {
 
   const handleSubmit = (event) => {
     event.preventDefault();
+    const fields = {
+      category,
+      createdAt,
+      amount,
+      currency,
+      note,
+      labels,
+    };
     if (!fields.category || !fields.createdAt || !fields.amount || !fields.currency) {
       console.log('Required fields');
     } else {
-      dispatch(postSpendingPending(fields));
+      if (spending) {
+        fields._id = spending._id; //eslint-disable-line
```

Comment:
Why you set id to the fields? You use `PATCH` method on back end site. It means that to update entity we don't need to provide a full list of object properties (only that changed).